app/actions/CommonActions.scala

```diff
@@ -8,11 +9,14 @@ import scala.concurrent.Future
 trait CommonActions {
 
   import CommonActions._
+
   val noCacheAction = resultModifier(noCache)
```

Comment:
Small style nit but I think the style guide says "Constant names should be in upper camel case", ie `NoCacheAction` etc:

http://docs.scala-lang.org/style/naming-conventions.html#objects